mypy/types.py

```diff
@@ -1457,8 +1458,9 @@ class TypeType(Type):
     # a generic class instance, a union, Any, a type variable...
     item = None  # type: Type
 
-    def __init__(self, item: Union[Instance, AnyType, TypeVarType, TupleType, NoneTyp,
-                                   CallableType], *, line: int = -1, column: int = -1) -> None:
+    def __init__(self, item: Bogus[Union[Instance, AnyType, TypeVarType, TupleType, NoneTyp,
+                                         CallableType]], *,
+                 line: int = -1, column: int = -1) -> None:
```

Comment:
I am a bit surprised the `Bogus` is needed here, and not in `make_normalized`, where the actual hack happens. But most probably I am just missing something.

Reply:
The crash does indeed happen in `make_normalized`, when it tries to coerce the arguments to the type required by `__init__`